recipes/ffnet/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "ffnet" %}
+{% set version = "0.8.3" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "1aaf35158239cd1e6f63504e884683abac87c77919af36cbefbf1169fed5d1a2" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
```

Comment:
I think you need `setuptools` and `toolchain` in the `build` section. Also is `networkx` required to build the package or just run it. If it's the latter, please delete it from the `build` list.